scripts/release_tools/import_prod.sh

```diff
@@ -33,4 +33,5 @@ argo submit ../../orchestration/workflows/prod/run-import-hca-total.yaml \
      -p source-bucket-prefix="$SOURCE_BUCKET_PREFIX" \
      -p staging-bucket-prefix="$STAGING_BUCKET" \
      -p dataset-id="d30e68f8-c826-4639-88f3-ae35f00d4185" \
-     -p data-repo-name="datarepo_hca_prod_20201120_dcp2"
+     -p data-repo-name="datarepo_hca_prod_20201120_dcp2" \
+     -p data-repo-project="broad-datarepo-terra-prod-hca2"
```

Comment:
Is this what we originally had hardcoded in the `jadeData` configmap?

Reply:
@quazi-broad yup